src/components/Cashout/Components/Initial.view.tsx

```
@@ -77,11 +77,15 @@ export const InitialCashoutView = ({
     }
 
     const isBelowMinLimit = useMemo(() => {
-        return !usdValue || parseFloat(usdValue) < MIN_CASHOUT_LIMIT
+        if (!usdValue) return false
+        const numericValue = parseFloat(usdValue)
+        return !isNaN(numericValue) && numericValue < MIN_CASHOUT_LIMIT
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)